modules/tinymce/src/core/main/ts/init/Render.ts

```diff
@@ -45,9 +50,29 @@ const loadLanguage = (scriptLoader: ScriptLoader, editor: Editor) => {
   }
 };
 
-const loadTheme = (scriptLoader: ScriptLoader, editor: Editor, suffix: string): Promise<void> => {
+/**
+ * Initiate the download of both theme and model in parallel, then wait for both.
+ *
+ * Each has their own special case where nothing needs to be waited for.
+ */
+const loadThemeAndModel = (scriptLoader: ScriptLoader, editor: Editor, suffix: string): Promise<void> => {
   const theme = Options.getTheme(editor);
 
+  // Special case the 'wait for model' code if RTC is loading, as it will provide a model instead
+  let waitForModel: () => Promise<void>;
+  if (!Rtc.isRtc(editor)) {
+    const model = Options.getModel(editor);
+    const modelUrl = Options.getModelUrl(editor);
+    const url = Type.isString(modelUrl) ? editor.documentBaseURI.toAbsolute(modelUrl) : `models/${model}/model${suffix}.js`;
```

Comment:
I know this is what the theme does, but this seems incorrect... it should be relative to the TinyMCE base url, not the document base url. That's probably a bug in the theme loading code as well.

Reply:
Yeah, if there's a bug they both need to change 🤔 I can see arguments either way, for a single-page application it might make more sense to the developer for the URL they provide to be relative to the page they are editing. Not the TinyMCE script.

Reply:
I think it's this way since you normally using script, css etc import relative to the page and your custom things are likely to live outside the tinymce directory so at the time it made sense to make them relative to the page not the tinymce script. And when loading from cloud it make no sense to have it relative to our cloud url. But the models here is a bit special we are not likely wanting third party to provide models so maybe they should just absolute urls since the only use case I can think of for us using a custom one is when you do cross project development or something. Wanting to load say the dom core from webpack into a project like powerpaste or something.